front_end/.browserslistrc

```diff
@@ -0,0 +1,4 @@
+> 1%
```

Comment:
I'm not sure how important this is but you may want to check what Google policy is regarding supported browsers (if you didn't of course).

Reply:
I did some research on this but couldn't come to a conclusion. Created and issue for this: #12